pkg/feature/manifest.go

```diff
@@ -14,6 +14,12 @@ import (
 //go:embed templates
 var embeddedFiles embed.FS
 
+const (
+	BaseDir        = "templates"
+	ServiceMeshDir = BaseDir + string(filepath.Separator) + "servicemesh"
```

Comment:
```suggestion
	ServiceMeshDir = path.Join(BaseDir, "servicemesh")
```

Reply:
Yes... I am tired :) Thank you, fixed in d873c21